probe/main.go

```diff
@@ -92,6 +94,9 @@ func main() {
 			case <-pubTick:
 				publishTicks.WithLabelValues().Add(1)
 				r.Host = hostTopology(hostID, hostName)
+				if dockerTagger != nil {
```

Comment:
I think this should be decoupled from dockerTagger; people will want this info even when not running on docker.